### ISSUE TITLE
[Portfolio Display] Bitcoin missing after sending from ledger to same ledger (different accounts)?

### ISSUE THREAD

**Issue by GittyGitGitGitty** [LedgerHQ/ledger-live-desktop#4989]
#### Ledger Live Version and Operating System

- Tested on **Ledger Live Desktop 2.41.3 / Ledger Nano X**
- Platform and version: **Arch Linux / AUR** https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=ledger-live

#### Expected behavior

Sending from 1st native segwit wallet to 2nd native segwit wallet on the same ledger device should keep the total balance the same because I am sending to myself. Both wallet balances should also be correct.

#### Actual behavior

Sending from 1st native segwit wallet to 2nd native segwit wallet on the same ledger device made approximately 0.025 bitcoin disappear from the 1st wallet (it's NOT a part of the tx). The 1st wallet's balance is now incorrect.

Connecting ledger to a freshly installed Ledger Live Desktop (with different user to avoid cache or previous settings) shows identical incorrect balance. Trying to send from the 1st wallet shows the incorrect balance available in the wallet.

The missing amount is also not recorded in the Ledger Live Desktop history or CSV.

#### Steps to reproduce the behavior

The setup:

- Ledger Nano X
- Two different native segwit wallets, both on same device
- Other Bitcoin wallets are on the device that aren't native segwit but they weren't involved in this tx as far as I know

What happened:

1. I funded the 1st wallet (it had some bitcoin already in it). The 2nd wallet was empty.
2. I sent a very small tx from the 1st wallet to the 2nd wallet. It went through perfect without problems.
3. I then sent a large tx from the 1st wallet to the 2nd wallet to the same address (I did not click `Receive` to get a new address, I reused the previous address). It failed repeatedly with `API Error 400` but it eventually went through.
4. During the send, the ledger live total balance / amount chart was strangely flickering between the correct amount and incorrect amount. It eventually settled on the incorrect amount.
5. The 2nd wallet balance (receiving wallet) is correct but the 1st wallet balance (sending wallet) is incorrect.

All balances were accurate before the final send. Does this have anything to do with change addresses?

**I have no access to the remaining balance when I try to send! Please help!**

**Comment by GittyGitGitGitty** [comment]
Clearing cache does not fix the issue.

**Comment by GittyGitGitGitty** [comment]
I don't think this is related to the Desktop versus Mobile display amount bug, but I could be wrong. I don't have access to the missing amount, period.

**Comment by GittyGitGitGitty** [comment]
Removing and adding the account in ledger live desktop does not fix the issue.

**Comment by GittyGitGitGitty** [comment]
Did I lose bitcoin because of this?

https://support.ledger.com/hc/en-us/articles/5129526865821-Solving-API-HTTP-400-error?support=true

**Comment by GittyGitGitGitty** [comment]
UPDATE: the issue *may* have been that Ledger Live was previously reporting the **wrong** portfolio balance and that sending this tx made the balance correct! I was basing my numbers off of Ledger Live's records but after looking at personal records and using Electrum to verify, this *could* the issue instead.

**Comment by gre** [comment]
Hi, thanks a lot for raising this issue. 

Unfortunately we can't investigate such cases on Github. Could you contact our official support  at https://support.ledger.com/  🙏  Thanks

**Comment by GittyGitGitGitty** [comment]
Hi @gre . Okay thanks for letting me know.

UPDATE:

I've fully confirmed that the issue was 100% related to a display bug and **only** a display bug.

After doing more research, it actually settled on the correct amount but the flashing gave the incorrect balance because it briefly accounted for the spend amount *twice*, thus leading to my confusion of the previous balance and me thinking the previous balance was wrong.

In short: a portfolio balance display bug!

I have access to the complete balance! No more alarm bells.
